src/styles/main.less

```diff
@@ -1,6 +1,7 @@
 @import '~bootstrap/less/variables.less';
 @import '~patternfly/dist/less/color-variables.less';
 @import '~patternfly/dist/less/variables.less';
+@import "../../app/styles/variables.less";
```

Comment:
We should be moving the variables we need under `src`. We shouldn't import from `app` to `src` since the app directory is meant to only be for the sample app.